packages/server/graphql/mutations/helpers/removeUserFromMeetingStages.ts

```diff
@@ -24,11 +28,19 @@ const removeUserFromMeetingStages = async (
         for (let i = 0; i < stages.length; i++) {
           const stage = stages[i]
           const {readyToAdvance} = stage
-          if (!readyToAdvance) continue
           const userIdIdx = readyToAdvance.indexOf(userId)
-          if (userIdIdx === -1) continue
-          readyToAdvance.splice(userIdIdx, 1)
-          isChanged = true
+          if (userIdIdx !== -1) {
+            readyToAdvance.splice(userIdIdx, 1)
+            isChanged = true
+          }
+          const scores = (stage as EstimateStage).scores
```

Comment:
let's rewrite this to:

```ts
if (stage.phaseType === 'ESTIMATE') {
```

that way, we aren't doing any casting & we know that `scores` exists.